setup.py

```diff
@@ -46,9 +46,10 @@
                  'same communication protocol: OSP (Open Scanner '
                  'Protocol)'),
     long_description=long_description,
+    long_description_content_type='text/markdown',
 
     # The project's main homepage.
-    url='http://www.openvas.org',
+    url='https://community.greenbone.net/t/gvm-10-beta/208',
```

Comment:
Maybe link to https://community.greenbone.net/t/about-the-open-scanner-protocol-osp-category/98 instead as the current used link will get outdated sooner then later?

Reply:
Or https://github.com/greenbone/ospd
but perhaps better leave as is for the time being.